src/views/PaymentPageView.vue

```diff
@@ -56,11 +102,47 @@ export default {
 </script>
 
 <template>
-  <PaymentConfirmed v-if="paymentSuccessful"></PaymentConfirmed>
-  <AwaitingPayment v-if="awaitingPayment" :qrCodeUrl="paymentQrCodeUrl"></AwaitingPayment>
-  {{ param1 }}
-  {{ param2 }}
-  {{ param3 }}
+  <div class="mx-auto text-center">
+    <div class="status-bar py-2">
+      <span class="text">{{statusMessage}}</span>
+    </div>
+    <ErrorPage v-if="!paymentSuccessful && errorMessage.length > 0" :errors="errorMessage"></ErrorPage>
+    <h1 v-else class="text-gray-200 font-bold py-4">
+      Lightning
+      <img src="@/assets/lightning-amber.png" alt="Lightning" class="inline-block mx-2 w-7">
+      Payment
+    </h1>
+    <PaymentConfirmed
+      v-if="paymentSuccessful"
+      :timeStamp="paymentTimeStamp"
+      :paymentAmount="amountCents"
+      :referenceId="referenceId"
+    ></PaymentConfirmed>
+    <AwaitingPayment v-if="awaitingPayment" :qrCodeUrl="paymentQrCodeUrl"></AwaitingPayment>
+    <div class="columns-auto">
+      <img src="@/assets/secure-payment-money-badger.png" alt="Secure Payment" class="mx-auto py-4 w-1/4"  />
+    </div>
+    <div :style="{ color: 'magenta', fontWeight: 'bold', textAlign: 'left' }">
```

Comment:
you can maybe just use display: none on this - it's useful debug info so having it easily on hand could be useful